README.md

```diff
@@ -6,31 +6,47 @@ Clone this repository and navigate into it.
 
 To use Docker to start all services, run
 
-```
+```sh
 docker compose up
 ```
 
 Then visit <http://localhost:8000> to see the site.
 
 To reach a shell within the Docker container, run
 
-```
+```sh
 docker compose run -it --remove-orphans web sh
 ```
 
 Import documents with this command:
 
-```
+```sh
 python manage.py import
 python manage.py import "test_docs/*"
 ```
 
-From there, to use a Python environment, run
+From there, to use a Python environment, run:
 
-```
+```sh
 python manage.py shell
 ```
 
+Run tests with the following commands:
+
+```sh
+# Run all tests
+docker compose run --it --remove orphans web python3 manage.py test
```

Comment:
I think the `-` character syntax needs adjustment here. There should be only one in `-it` and `remove-orphans` is hyphenated right?

Reply:
Genuinely have no idea how that happened. Yes.